lib/rspec/core/example.rb

```diff
@@ -225,10 +225,7 @@ def run(example_group_instance, reporter)
         rescue Exception => e
           set_exception(e)
         ensure
-          ExampleGroup.each_instance_variable_for_example(@example_group_instance) do |ivar|
-            @example_group_instance.instance_variable_set(ivar, nil)
-          end
-          @example_group_instance = nil
+          @example_group_instance = nil # if you love something... let it go
```

Comment:
I don't care either way.  Personally, I like when people put a little personality in their commits, so I'm happy to let it stay.